addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -128,7 +128,7 @@ return core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin, {
 
             if (do_update && (this.running_tour === name || (!this.running_tour && !tour.test && !tour_is_consumed))) {
                 this._to_next_step(name, 0);
-                this.update(name);
+                this.update(name, true);
```

Comment:
could you try removing this line, and call this.update (without tour_name) at the end of _register_all (in Promise.all(...).then())? With the current solution, update is called for each tour, and activates each time the first one.

Reply:
No need of the extra register_mode argument in this case

Reply:
It works fine but wouldn't it be enough to remove the name of the update method in register, it also works and looks faster?